pkg/resources/manifests/deploy.yaml

```diff
@@ -1,91 +1,99 @@
+---
+# This file is generated by 'make generate-manifests'; DO NOT EDIT.
 apiVersion: v1
 kind: Namespace
 metadata:
   name: gadget
 ---
+# Source: gadget/templates/serviceaccount.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: gadget
   namespace: gadget
 ---
+# Source: gadget/templates/clusterrole.yaml
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
In general, It might be problem like if `namespace` is moved in the end we will run into errors. But here we only swap `ClusterRole` and `Role` which aren't related so should be fine. Helm decide to order them like that as mentioned in this [section](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package).